### PR TITLE
Issue #3252050 by nkoporec: Ajax error when exporting event enrollments

### PR DESCRIPTION
**Pull request by nkoporec**
## Problem
When trying to export event enrolments an AJAX error occurs.

## Solution
The issue is that in UserExportPluginBase the method profileGetFieldValue is typed hinted to return a string, but that is not entirely true, because the field can be NULL. So we need to make sure we always return a string.

## Issue tracker
https://www.drupal.org/project/social/issues/3252050

From team Nucleus: https://getopensocial.atlassian.net/browse/TB-4575

## How to test
1. Install the social_event_enrollments_export and social_birthday
2. Go to the event with enrollments
3. Export the enrollments
4. Please also enable social_event_an_enroll and social_event_an_enroll_enrolments_export
5. Enable anonymous enrollment in the event
6. Do some AN enrollment to the event with First & last name filled
7. Do some AN enrollment to the event with one of First & last names filled 
8. Export all user's data.

## Screenshots
N.A

## Release notes
Fixed an issue with exporting event enrolments.

## Change Record
N.A

## Translations
N.A

### CODE REVIEW

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2631-xcnl2hgmeoxchqzlvsebkjlyanb74frn.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61a77010b880b2370a0eae05

**Comment by NickDaelemans** [comment]
Looks good!

**Review by NickDaelemans — APPROVED** [review]

**Review by NickDaelemans — APPROVED** [review]

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2631-xcnl2hgmeoxchqzlvsebkjlyanb74frn.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61a77010b880b2370a0eae05

File: modules/social_features/social_event/modules/social_event_managers/src/Plugin/views/field/SocialEventManagersViewsBulkOperationsBulkForm.php [hunk 1]
```diff
@@ -347,17 +347,15 @@ public function getEntityLabel(EntityInterface $entity) {
...
+    // It must be a Guest, so we pick the name values we can get.
     if (!$profile) {
       $name = '';
-      $first_name = $entity->get('field_first_name')->getValue()[0]['value'];
-      $last_name = $entity->get('field_last_name')->getValue()[0]['value'];
-      if (!empty($first_name)) {
-        $name .= $first_name;
+      if (!empty($first_name = $entity->get('field_first_name')->getValue())) {
+        $name .= $first_name[0]['value'];
         $name .= ' ';
```
Comment (Ressinel) [inline modules/social_features/social_event/modules/social_event_managers/src/Plugin/views/field/SocialEventManagersViewsBulkOperationsBulkForm.php:355]: If `field_last_name` is empty we will have a name with space on the right side. I'd recommend `trim` the name before return.

**Review by Ressinel — CHANGES_REQUESTED** [review]

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2631-xcnl2hgmeoxchqzlvsebkjlyanb74frn.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61a77010b880b2370a0eae05

**Review by Ressinel — APPROVED** [review]

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2631-xcnl2hgmeoxchqzlvsebkjlyanb74frn.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61a77010b880b2370a0eae05

**Pull request merged**